packages/foundations/scripts/color-placeholders-generator.js

```diff
@@ -159,22 +171,32 @@ exports.generateColorUtilitityPlaceholder = (colorToken) => {
 			if (colorToken[value].enabled) {
 				// Text & elements & border
 				output += `
+%${prefix}-${value}-component-interactive-state {
+	&:hover{
+		background-color: var(--${prefix}-${value}-hover,
+		#{$${prefix}-${colorToken[value].hover.name}});
+	}
+	&:active{
+		background-color: var(--${prefix}-${value}-pressed,
+		#{$${prefix}-${colorToken[value].pressed.name}});
+	}
+}
+
 %${prefix}-${value}-component-ia {
 	--db-current-color: var(--${prefix}-${value}-on-enabled,
 	#{$${prefix}-${colorToken[value].on.enabled.name}});
     color: var(--${prefix}-current-color);
 	background-color: var(--${prefix}-${value}-enabled,
 	#{$${prefix}-${colorToken[value].enabled.name}});
-	&:enabled {
-		&:hover{
-			background-color: var(--${prefix}-${value}-hover,
-			#{$${prefix}-${colorToken[value].hover.name}});
-		}
-		&:active{
-			background-color: var(--${prefix}-${value}-pressed,
-			#{$${prefix}-${colorToken[value].pressed.name}});
+	&:is(#{$interactive-elements}) {
+		&:enabled{
+			@extend %${prefix}-${value}-component-interactive-state;
 		}
 	}
+
+	&:is(a) {
+		@extend %${prefix}-${value}-component-interactive-state;
+	}
```

Comment:
```suggestion
	&a,:is(#{$interactive-elements}):enabled{
		@extend %${prefix}-${value}-component-interactive-state;
	}
```

why couldn't you merge this with the other selector above?